src/Tribe/Main.php

```diff
@@ -115,6 +115,8 @@ public function __construct( $context = null ) {
 		$this->doing_ajax = defined( 'DOING_AJAX' ) && DOING_AJAX;
 
 		Tribe__Extension_Loader::instance();
+		Tribe__Deprecation::instance();
```

Comment:
Shouldn't we use `tribe_singleton()` to set this up?

Reply:
Eventually I'm sure we should. It was already setup to be a singleton thought and had an `instance()` method. My thinking is we could ticket that rather than trying to squeeze it into an over budget ticket minutes before the smoketest starts.